dids/src/main/kotlin/web5/sdk/dids/DidMethod.kt

```diff
@@ -64,121 +32,3 @@ public interface CreateDidOptions
  * ```
  */
 public interface CreationMetadata
```

Comment:
Looks like `CreationMetadata` is only used with did:ion, which we're dropping support for anyway, right? Is it worth dropping ion in this PR?

Reply:
aw yea i'm deleting this, ty!